### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlexShubin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+## Intro
+
+Workshops are designed to give developers who have no experience with Swift an overview of modern iOS-apps development and share with more experienced iOS-developers knowledge about unidirectional architectures and particularly [Composable Architecture](https://github.com/pointfreeco/swift-composable-architecture).
+
+Workshops require professional programming knowledge. If you had no experience with Swift it's ok. You can drop any time they get harder. And vice versa you can join any time checking out the latest commit in this repo.
+
+## Workshops plan
+
+We'll be building slightly modified Babbel's code challenge. Here's the app interface:
```
Comment (GLTaylor) [inline README.md]:
```suggestion
We'll be building a slightly modified version of Babbel's code challenge. Here's the app interface:
```

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
-# ArchitechtureWorkshops
\ No newline at end of file
+# ArchitechtureWorkshops
+
+## Intro
+
+Workshops are designed to give developers who have no experience with Swift an overview of modern iOS-apps development and share with more experienced iOS-developers knowledge about unidirectional architectures and particularly [Composable Architecture](https://github.com/pointfreeco/swift-composable-architecture).
+
+Workshops require professional programming knowledge. If you had no experience with Swift it's ok. You can drop any time they get harder. And vice versa you can join any time checking out the latest commit in this repo.
```
Comment (GLTaylor) [inline README.md]:
```suggestion
Workshops require professional programming knowledge. If you have no experience with Swift it's okay. You can drop if at any time they get harder. And vice versa; you can join any time by checking out the latest commit in this repo.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+ 3. Introducing the app state and sharing it between different scenes
+ 4. Introducing the app state reducer
+ 5. Building and testing service layer
+ 6. Introducing side-effects and Composable Architecture
+ 7. Introducing Reducers composition, using Composable Architecture to modularize the app
+ 8. Testing the app modules
+
+## Setup
+
+Setup the latest version of XCode from the AppStore. Make sure you ran it at least once in order to install "Additional Components" (you'll be guided).
```
Comment (pooja-salpekar) [inline README.md]: Should we link to the latest version on the App Store or mention the version number?

**Review by pooja-salpekar — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (mAu888) [inline README.md]:
```suggestion
Setup the latest version of Xcode from the AppStore. Make sure you ran it at least once in order to install "Additional Components" (you'll be guided).
```

**Pull request merged**